install-wings.sh

```diff
@@ -458,8 +458,8 @@ letsencrypt() {
 perform_install() {
   echo "* Installing pterodactyl wings.."
   [ "$OS" == "ubuntu" ] || [ "$OS" == "debian" ] && apt_update
```

Comment:
Are you sure this works as expected? I don't think this is equivalent to

```bash
if [ "$OS" == "ubuntu" ] || [ "$OS" == "debian" ]; then
    apt_update
fi
```

but correct me if I am wrong.

Reply:
```bash
false || true && echo "Works"
true || false && echo "Works"
false || false && echo "Doesn't work"
true || true && echo "Works"
```
```bash
Works
Works
Works
```
Based on this it works. But I would also be skeptic about this if I wouldn't have tested it before.